doc/ranger.pod

```diff
@@ -225,6 +225,15 @@ This only works in terminology. It can render vector graphics, but works only lo
 
 To enable this feature, set the option C<preview_images_method> to terminology.
 
+=head3 ueberzug
+
+Überzug is a command line utility which draws images on terminals using child
+windows. It requires PIL (or pillow) and relies on X11. This makes it
+compatible (in a limited way, i.e., tmux splits are not supported) with many
```

Comment:
Very minor: I don't think there should be a comma after "i.e."

Reply:
I adhere to the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html) on this.

Reply:
TIL